quicklens/src/main/scala-2.13+/com.softwaremill.quicklens/package.scala

```diff
@@ -174,11 +174,33 @@ package object quicklens {
     }
   }
 
-  implicit def optionQuicklensFunctor[A]: QuicklensFunctor[Option, A] =
-    new QuicklensFunctor[Option, A] {
+  implicit def optionQuicklensFunctor[A]: QuicklensFunctor[Option, A] with QuicklensGetFunctor[Option, A] =
+    new QuicklensFunctor[Option, A] with QuicklensGetFunctor[Option, A] {
       override def map(fa: Option[A])(f: A => A) = fa.map(f)
+      override def get(fa: Option[A])(f: A => A) = Some(fa.map(f).get)
     }
 
+  implicit def optionQuicklensFunctor[A]
+  : QuicklensFunctor[Option, A] with QuicklensSingleAtFunctor[Option, A] =
+    new QuicklensFunctor[Option, A] with QuicklensSingleAt[Option, A] {
+      override def map(fa: Option[A])(f: A => A) = fa.map(f)
+      override def at(fa: Option[A])(f: A => A) = Some(fa.map(f).get)
+      override def atOrElse(fa: Option[A], default: => A)(f: A => A): Option[A] = fa.orElse(Some(default)).map(f)
+    }
+
+  trait QuicklensSingleAtFunctor[F[_], T] {
+    def at(fa: F[T])(f: T => T): F[T]
+    def atOrElse(fa: F[T], default: => T)(f: T => T): F[T]
+  }
+
+  implicit class QuicklensSingleAt[F[_], T](t: F[T])(implicit f: QuicklensSingleAtFunctor[F, T]) {
```

Comment:
Open to a better name here.

Reply:
yeah, I feel the same need for a better name, but can't come up with anything. So single it is :)